### PR TITLE
Fix casing of npm: NPM > npm

### PR DESCRIPTION
**Pull request by wooorm**
I kept the two ([1](https://github.com/travis-ci/travis.rb/blob/89cb71786453cd07d9248ee31c0e135e23541ed4/lib/travis/cli/setup.rb#L16), [2](https://github.com/travis-ci/travis.rb/blob/89cb71786453cd07d9248ee31c0e135e23541ed4/lib/travis/cli/setup/npm.rb#L7)) non-message references to the wrongly cased npm as they were.  I can change those too, though.

> ## 13: Always be lowercase
> 
> If you see npm referred to as “NPM” anywhere except right there in this sentence, then you know that I didn’t write it. “NPM” is the National Association of Pastoral Musicians. If you need some church music, use NPM. If you want node packages installed, then you need npm.

— “[10 Cool Things You Probably Didn’t Realize npm Could Do](http://blog.izs.me/post/1675072029/10-cool-things-you-probably-didnt-realize-npm)”, by Isaac Z. Schlueter (npm's creator).

### CODE REVIEW

File: assets/travis.sh [hunk 1]
```diff
@@ -21,7 +21,7 @@ if type compdef 1>/dev/null 2>/dev/null; then
...
...omatic deployment to Appfog" artifacts:"automatic artifact shipping to S3" biicode:"automatic publish to biicode" cloudcontrol:"automatic deployment to cloudControl" cloudfiles:"automatic pushing to Rackspace Cloud Files" cloudfoundry:"automatic deployment to Cloud Foundry" cloud66:"Automatic deployment to Cloud 66" codedeploy:"triggering a new deployment on Amazon CodeDeploy" deis:"automatic deployment to a deis app" divshot:"deployment to Divshot.io" elasticbeanstalk:"deployment to AWS Elastic Beanstalk" engineyard:"automatic deployment to Engine Yard" gcs:"automatic pushing to Google Cloud Storage" hackage:"automatic deployment of hackage packages" heroku:"automatic deployment to Heroku" modulus:"deployment to Modulus" npm:"automatic release to npm" ninefold:"Automatic deployment to Ninefold" nodejitsu:"automatic deployment to Nodejitsu" openshift:"automatic deployment to OpenShfit" opsworks:"deployment to OpsWorks" pypi:"automatic deployment to PyPI" releases:"Upload Assets to GitHub Releases" rubygems:"automatic release to RubyGems" s3:"automatic pushing to S3" sauce_connect:"Sauce Connet addon for Sauce Labs integration") _describe -t common-commands 'common commands' list; }
```
Comment (BanzaiMan) [inline assets/travis.sh:24]: Hello, there. Sorry for the long delay.

If you are interested in this PR, please remove this unrelated change. If not, I can do the work while keeping credit to you.

Comment (wooorm) [reply assets/travis.sh:24]: 👋

It’s not unrelated: it also changes `NPM` > `npm`.

Comment (BanzaiMan) [reply assets/travis.sh:24]: oh, sorry!

Comment (BanzaiMan) [reply assets/travis.sh:24]: Please resolve the conflict, and we'll move forward.

Comment (wooorm) [reply assets/travis.sh:24]: Done!

**Pull request merged**